vimrc

```diff
@@ -57,16 +67,11 @@ augroup vimrcEx
 
   " Allow stylesheets to autocomplete hyphenated words
   autocmd FileType css,scss,sass setlocal iskeyword+=-
-augroup END
 
-" Softtabs, 2 spaces
-set tabstop=2
-set shiftwidth=2
-set shiftround
-set expandtab
-
-" Display extra whitespace
-set list listchars=tab:»·,trail:·,nbsp:·
+  let g:go_fmt_command = "goimports"
```

Comment:
These are settings provided to customize `vim-go`. As far as I know everyone who's set up their personal versions of these has this setting.